Makefile

```diff
@@ -315,18 +315,21 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
 catalog-build: bundle-push opm ## Generate operator-catalog dockerfile using the operator-bundle image built and published above; then build catalog image
-	$(OPM) index add --container-tool $(CONTAINER_ENGINE) --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT) --generate
-	$(CONTAINER_ENGINE) build --platform $(PLATFORM) -f index.Dockerfile -t $(CATALOG_IMG) --label $(LABEL) .
+    ## [GA] added '-d docker/index.Dockerfile' to avoid generating in the root
+	$(OPM) index add --container-tool $(CONTAINER_ENGINE) --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT) --generate -d docker/index.Dockerfile
+	$(CONTAINER_ENGINE) build --platform $(PLATFORM) -f docker/index.Dockerfile -t $(CATALOG_IMG) --label $(LABEL) .
+	## remove Sqlite db
+	rm -fr database
 
 .PHONY: catalog-push
 catalog-push: ## Push catalog image to registry
-	$(MAKE) operator-push IMG=$(CATALOG_IMG)
+	$(MAKE) image-push IMG=$(CATALOG_IMG)
 
 .PHONY:
-release-build: bundle operator-build bundle-build catalog-build ## Build operator, bundle + catalog images
+release-build: bundle image-build bundle-build catalog-build ## Build operator, bundle + catalog images
 
 .PHONY:
-release-push: operator-push bundle-push catalog-push ## Push operator, bundle + catalog images
+release-push: image-push bundle-push catalog-push ## Push operator, bundle + catalog images
```

Comment:
this doesn't really track, name-wise. 

"image-push" may be a reusable target but in this context - called without params - it should be "operator-push", since all three (operator, bundle and catalog) are images.

Reply:
I agree it is not ideal but better than "operator-", this image is not an operator, it is a manager with controller(s) for operator (that's why operator SDK did not choose it I guess). The most technically correct is "manager-" but I doubt it is clear for everyone. 
TBH, I see no problem with old "docker" (even if it is podman :) ) docker-push and docker-build are very known tags for operator developers.

Reply:
in the context of downstream terminology, the operator/manager/controller container is called "operator" and its csv/manifest/metadata content is called "operator-bundle".

But sure, if you prefer `image-` we can do that... I almost did the same a couple weeks ago when I refactored things in https://github.com/janus-idp/operator/commit/4ba6407bdaa906f1420962558c0ba2e70c8eaf6b#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R146-R150 to remove the duplicate stages for `docker-` and `podman-`.

Reply:
@nickboldt Like I said I do not "love" 'image-' option. 
It was rather a compromise with respect to yours "downstream" preferences (I understand how much you do not like "docker-"). 
In general, I do not see a big reason renaming it from defaults at all TBH.